modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js

```diff
@@ -92,6 +92,27 @@ asset.renderer = function(ctx) {
                     }
                 }
             },
+            permissionMetaDataPopulator: function(page, util) {
+                var ptr = page.leftNav || [];
+                var am = assetManager(ctx.session,ctx.assetType);
+                var entry;
+                var allowedPages = ['details','lifecycle','update','associations','permissions'];
+                log.debug('Permission populator ' + page.meta.pageName);
+                if(allowedPages.indexOf(page.meta.pageName)>-1){
```

Comment:
Thanks for pointing it out. Fixed with the following PR - https://github.com/wso2/product-greg/pull/851